emperor/_pandas.py

```diff
@@ -0,0 +1,103 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, emperor development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE.md, distributed with this software.
+# ----------------------------------------------------------------------------
+from __future__ import division
+
+import numpy as np
+import pandas as pd
+
+from emperor.core import Emperor
+from skbio import OrdinationResults
+
+
+def scatterplot(df, x=None, y=None, z=None, remote=True):
+    """Create an Emperor scatter plot from a Pandas DataFrame
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        Pandas DataFrame with the data to display, this includes both
+        *metadata* and coordinates to position the samples in a 3D space.
```

Comment:
It contains both metadata and coordinates within the same dataframe.  Right
@ElDeveloper?

On Mon, Jul 11, 2016 at 12:21 PM, Antonio Gonzalez <notifications@github.com
> wrote:

> In emperor/_pandas.py
> <https://github.com/biocore/emperor/pull/517#discussion_r70319694>:
>
> > +
> > +import numpy as np
> > +import pandas as pd
> > +
> > +from emperor.core import Emperor
> > +from skbio import OrdinationResults
> > +
> > +
> > +def scatterplot(df, x=None, y=None, z=None, remote=True):
> > +    """Create an Emperor scatter plot from a Pandas DataFrame
> > +
> > +    Parameters
> > +    ----------
> > +    df : pd.DataFrame
> > +        Pandas DataFrame with the data to display, this includes both
> > +        *metadata* and coordinates to position the samples in a 3D space.
>
> why *metadata* and not *coordinates*?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/biocore/emperor/pull/517/files/508540e0df1c5599072fc3a73848b2e9ff3e7795#r70319694>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AD_a3XtOnAXkG5C1FboNxpgY0P5_e8zeks5qUpe_gaJpZM4JJDNq>
> .
>